packages/plugin-aws-sdk/test/sqs.spec.ts

```diff
@@ -132,7 +139,10 @@ describe('sqs', () => {
     });
 
     describe('process spans', () => {
-        beforeEach(() => plugin.enable(AWS, provider, logger));
+        beforeEach(() => {
+            instrumentation.disable();
```

Comment:
what is the motivation?

Reply:
the motivation is to get it to work, it doesn't work without this because we're creating mocks in the top level for-each.